pkg/metricscollector/v1beta1/common/const.py

```diff
@@ -20,6 +20,9 @@
 DEFAULT_WAIT_ALL_PROCESSES = "True"
 # Default value for directory where TF event metrics are reported
 DEFAULT_METRICS_FILE_DIR = "/log"
+# Default value for directory where TF event metrics are reported
+DEFAULT_METRICS_FILE_KFPV1_DIR = "/tmp/outputs/mlpipeline_metrics"
```

Comment:
Can we reuse `DEFAULT_METRICS_FILE_DIR`? Does this define in KFP v1 as a default value?

Reply:
In KFP v1 this is a default path where output metrics are saved in Python operators.

Reply:
sgtm.